pkg/config/server/server.go

```diff
@@ -89,7 +89,12 @@ func DefaultConfig() Config {
 // config struct with merged configuration from all provided files.
 func ParseConfigFiles(files []string) (Config, error) {
 	c := DefaultConfig()
-	return c, cfgutil.ParseYAML(&c, DefaultConfig(), files...)
+	err := cfgutil.ParseYAML(&c, DefaultConfig(), files...)
+	if err == nil && c.Backup.LocalDC == "" {
+		c.Backup.LocalDC = c.Database.LocalDC
```

Comment:
I wanted to add `localDC` field (that already exists in database config) to backup config. 
I figured out that it might cause an inconvenience for the user to specify `localDC` in both backup and database `yaml` files, so if possible, I wanted to copy `localDC` from database to backup config.

Maybe I should just take away `'yaml:"local_dc"'` tag from backup config and copy it blindly from database config?

Reply:
As discussed during the meeting, localDC is no longer needed for restore purposes, so I am dropping this commit.

Reply:
I guess it's not commit yet. I'll resolve the conversation when it's done.